src/server/game/AI/CoreAI/CombatAI.cpp

```diff
@@ -55,32 +55,36 @@ void AggressorAI::UpdateAI(uint32 /*diff*/)
 void CombatAI::InitializeAI()
 {
     for (uint32 spell : me->m_spells)
+    {
```

Comment:
No to brackets here - the inner `if` is just a single statement, we only require them in case there is more than 1 statement in nested ifs or there is `else`